scripts/ubuntu-jammy-from-community-gui/06-wayland_errrata.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -exv
+
+# init helpers
+helpers_dir=${MONOPACKER_HELPERS_DIR:-"/etc/monopacker/scripts"}
+for h in ${helpers_dir}/*.sh; do
+    . $h;
+done
+
+# see https://github.com/mozilla-platform-ops/monopacker/issues/138
+
+
+#
+# dconf settings
+#
+
+# /etc/dconf/profile/user should have (no leading spaces):
+#
+#   user-db:user
+#   system-db:local
+cat > /etc/dconf/profile/user << EOF
+user-db:user
+system-db:local
+EOF
+
+# /etc/dconf/db/local.d/00-tc-gnome-settings should have (no leading spaces): 
+#
+# # /org/gnome/desktop/session/idle-delay
+# [org/gnome/desktop/session]
+# idle-delay=uint32 0
+#
+# # /org/gnome/desktop/lockdown/disable-lock-screen
+# [org/gnome/desktop/lockdown]
+# disable-lock-screen=true
+cat > /etc/dconf/db/local.d/00-tc-gnome-settings << EOF
+# /org/gnome/desktop/session/idle-delay
+[org/gnome/desktop/session]
+idle-delay=uint32 0
+
+# /org/gnome/desktop/lockdown/disable-lock-screen
+[org/gnome/desktop/lockdown]
+disable-lock-screen=true
+EOF
+
+# make dbus read the new configuration
+sudo dconf update
+
+# test
+ls -hal /etc/dconf/db/
+
+
+#
+# gdm3 settings  
+#
+
+# in [daemon] block of /etc/gdm3/custom.conf we need:
+#
+# XorgEnable=false
+
+# TODO: how to do this? augeas or crudini?
+#  - https://superuser.com/questions/155418/is-there-a-program-script-to-modify-conf-files
+
+apt install -y crudini
+crudini  --set ./custom.conf daemon XorgEnable 'false'
+
+cat /etc/gdm3/custom.conf
+grep 'XorgEnable' /etc/gdm3/custom.conf
+
+
+#
+# gdm3 service file
+#
+
+# copy /lib/systemd/system/gdm3.service to /etc/systemd/system and change its 
+# ExecStartPre to `/bin/sleep 30`
+
+cp /lib/systemd/system/gdm3.service /etc/systemd/system/gdm3.service
+crudini --set /etc/systemd/system/gdm3.service Service ExecStartPre '/bin/sleep 30'
+
+cat /etc/systemd/system/gdm3.service
+grep 'ExecStartPre' /etc/systemd/system/gdm3.service
+
+
+#
+# extra packages
+#
+
+apt install -y ubuntu-restricted-extras wl-clipboard
+
+
+#
+# TODO: generate mutter's monitors.xml (only if needed)
```

Comment:
it's needed, I just dont know exactly what we need to set for some values (`connector`, `vendor`, `product`, `serial`), e.g., mine on my laptop:
```
alex@portable-alex:~$ cat ~/.config/monitors.xml
<monitors version="2">
  <configuration>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <scale>1</scale>
      <monitor>
        <monitorspec>
          <connector>DP-5</connector>
          <vendor>AOC</vendor>
          <product>Q3279WG5B</product>
          <serial>0x00008ce6</serial>
        </monitorspec>
        <mode>
          <width>2560</width>
          <height>1440</height>
          <rate>59.951</rate>
        </mode>
      </monitor>
    </logicalmonitor>
    <logicalmonitor>
      <x>2560</x>
      <y>0</y>
      <scale>1</scale>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>DP-4</connector>
          <vendor>AOC</vendor>
          <product>Q3279WG5B</product>
          <serial>0x00000745</serial>
        </monitorspec>
        <mode>
          <width>2560</width>
          <height>1440</height>
          <rate>59.951</rate>
        </mode>
      </monitor>
    </logicalmonitor>
    <disabled>
      <monitorspec>
        <connector>eDP-1</connector>
        <vendor>CSO</vendor>
        <product>0x1407</product>
        <serial>0x00000000</serial>
      </monitorspec>
    </disabled>
  </configuration>
  <configuration>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <scale>1</scale>
      <monitor>
        <monitorspec>
          <connector>DP-5</connector>
          <vendor>IVM</vendor>
          <product>PL3293UH</product>
          <serial>1213432401132</serial>
        </monitorspec>
        <mode>
          <width>3840</width>
          <height>2160</height>
          <rate>59.997</rate>
        </mode>
      </monitor>
    </logicalmonitor>
    <logicalmonitor>
      <x>3840</x>
      <y>0</y>
      <scale>1</scale>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>DP-4</connector>
          <vendor>IVM</vendor>
          <product>PL3293UH</product>
          <serial>1213432401136</serial>
        </monitorspec>
        <mode>
          <width>3840</width>
          <height>2160</height>
          <rate>59.997</rate>
        </mode>
      </monitor>
    </logicalmonitor>
    <disabled>
      <monitorspec>
        <connector>eDP-1</connector>
        <vendor>CSO</vendor>
        <product>0x1407</product>
        <serial>0x00000000</serial>
      </monitorspec>
    </disabled>
  </configuration>
  <configuration>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <scale>1</scale>
      <monitor>
        <monitorspec>
          <connector>DP-7</connector>
          <vendor>IVM</vendor>
          <product>PL3293UH</product>
          <serial>1213432401132</serial>
        </monitorspec>
        <mode>
          <width>3840</width>
          <height>2160</height>
          <rate>59.997</rate>
        </mode>
      </monitor>
    </logicalmonitor>
    <logicalmonitor>
      <x>3840</x>
      <y>0</y>
      <scale>1</scale>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>DP-6</connector>
          <vendor>IVM</vendor>
          <product>PL3293UH</product>
          <serial>1213432401136</serial>
        </monitorspec>
        <mode>
          <width>3840</width>
          <height>2160</height>
          <rate>60.000</rate>
        </mode>
      </monitor>
    </logicalmonitor>
    <disabled>
      <monitorspec>
        <connector>eDP-1</connector>
        <vendor>CSO</vendor>
        <product>0x1407</product>
        <serial>0x00000000</serial>
      </monitorspec>
    </disabled>
  </configuration>
  <configuration>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <scale>1</scale>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>eDP-1</connector>
          <vendor>CSO</vendor>
          <product>0x1407</product>
          <serial>0x00000000</serial>
        </monitorspec>
        <mode>
          <width>3840</width>
          <height>2160</height>
          <rate>60.000</rate>
        </mode>
      </monitor>
    </logicalmonitor>
  </configuration>
</monitors>
```

Reply:
Should we just create one by hand and hope that the connector doesn't matter? How should we proceed?

Reply:
see below